src/network/clientpackethandler.cpp

```diff
@@ -1183,6 +1183,29 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 	m_client_event_queue.push(event);
 }
 
+void Client::handleCommand_HudSetClouds(NetworkPacket* pkt)
+{
+	std::string datastring(pkt->getString(0), pkt->getSize());
+	std::istringstream is(datastring, std::ios_base::binary);
```

Comment:
This is cargo cult programming, sorry :-/ I can rewrite it, but it will take time.